tests/helpers/main.ts

```diff
@@ -33,6 +36,17 @@ interface ZipNodeReturn extends ZipReturn {
   files: TestFunctionResult[]
 }
 
+let cleanupDirectories: string[] = []
```

Comment:
nit: You mention this in an inline comment, but having this in an actual comment in the code would be useful.

```suggestion
// Accumulator of temporary directories that need to be cleaned up afterwards.
let cleanupDirectories: string[] = []
```